tables/de-comp6.utb

```diff
@@ -0,0 +1,227 @@
+# liblouis: German 6 dots computer Braille table
+#
+# Copyright (C) 2022 SBS Schweizerische Bibliothek für Blinde, Seh- und Lesebehinderte
+#
+# This file is part of liblouis.
+#
+# liblouis is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 2.1 of the
+# License, or (at your option) any later version.
+#
+# liblouis is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with liblouis. If not, see
+# <http://www.gnu.org/licenses/>.
+
+#-name: Deutsches 6-Punkte Computerbraille
+#-index-name: German, computer, 6-dot
+#-display-name: German 6-dot computer braille
+#
+#+language:de
+#+type:computer
+#+contraction:no
+#+grade:0
```

Comment:
The contraction field shouldn't be included when the type is "computer".

I would also omit the grade field because grade 0 means "Basisschrift".

Reply:
should be fixed now